### DISCUSSION TITLE
Thank you for using SOLID principles

### DISCUSSION THREAD

**Discussion by theodoreOnzGit** [SpiceSharp/SpiceSharp#203] [General]
Just want to commend you for writing such nice readable code,
that as far as i can tell, uses SOLID principles. 

Especially when u use lots of interfaces, and don't have tightly coupled code spanning thousands of lines long,

it makes it easier to understand.

Thank you!

**Comment by svenboulanger** [comment]
Thanks for the compliment 👍 
Glad to hear you notice what I have probably been thinking about and changing most...
